src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -27,7 +27,7 @@ public static void CompileDeepTree_NoStackOverflow(bool useInterpreter)
             Assert.Equal(n, f());
         }
 
-        [Theory, ActiveIssue("https://github.com/dotnet/corefx/issues/15533"), ClassData(typeof(CompilationTypes))]
+        [Theory, ClassData(typeof(CompilationTypes))]
```

Comment:
Yep. I just came to the same conclusion reading over the issues linked off that. Windows has an underlying minimum of 64K (unless that's changed since I last dealt with it directly, which was several versions of Windows ago) so it was fine on those, but it's probably not worth making OS-specific when we do have the same thing covered in outerloop anyway. I'll delete the test completely.